.github/workflows/ci.yml

```diff
@@ -215,6 +214,24 @@ jobs:
           # was created in this workflow.
           tls_ca_cert: ${{ env.TLS_CA_CERT }}
 
+      - name: Cleanup out dir
+        run: sudo rm -rf out_repo
+
+      - name: Run Bindplane Action (Directory)
+        uses: ./
+        with:
+          bindplane_remote_url: https://${{ env.MAIN_IP }}:3001
+          bindplane_username: admin
+          bindplane_password: admin
+          # This is a directory
+          processor_path: test/resources/processors
+          # Should detect the resource in destinations/ and the
+          # two resources in the 'multi' sub directory
+          destination_path: test/resources/destinations
```

Comment:
Processor path is a path to a directory with three resource files. Destination path is a path to a directory with a resource file containing two destinations and a sub directory `multi` containing two individual destination resource files. A total of four destination resources.